spec/integration_spec.rb

```diff
@@ -22,12 +22,28 @@ def tables
 
   let(:direction) { :up }
 
-  before { ActiveRecord::Migration.verbose = false }
-
   it 'has a version number' do
     expect(PerconaMigrator::VERSION).not_to be nil
   end
 
+  context 'when the migration logging is enabled' do
+    before { ActiveRecord::Migration.verbose = true }
+
+    it 'sends the output to the stdout' do
+      expect($stdout).to receive(:puts).at_least(:once)
+      ActiveRecord::Migrator.new(direction, [migration_fixtures], 1).migrate
+    end
+  end
+
+  context 'when the migration logging is disabled' do
+    before { ActiveRecord::Migration.verbose = false }
```

Comment:
It's disable in the spec helper, so it applies more on the other context, but I'll restore both